airbyte-integrations/bases/standard-destination-test/src/main/java/io/airbyte/integrations/standardtest/destination/TestDestination.java

```diff
@@ -608,6 +610,51 @@ protected int getMaxRecordValueLimit() {
     return 1000000000;
   }
 
+  @Test
+  void testCustomDbtTransformations() throws Exception {
+    if (!implementsBasicNormalization()) {
+      return;
+    }
+
+    final JsonNode config = getConfigWithBasicNormalization();
+
+    final DbtTransformationRunner runner = new DbtTransformationRunner(pbf, NormalizationRunnerFactory.create(
+        getImageName(),
+        pbf,
+        config));
+    runner.start();
+    final Path transformationRoot = Files.createDirectories(jobRoot.resolve("transform"));
+    final OperatorDbt dbtConfig = new OperatorDbt()
+        .withGitRepoUrl("https://github.com/fishtown-analytics/jaffle_shop.git")
+        .withGitRepoBranch("main")
+        .withDockerImage("fishtownanalytics/dbt:0.19.1")
+        .withDbtArguments("debug");
+    if (!runner.run(JOB_ID, JOB_ATTEMPT, transformationRoot, config, dbtConfig)) {
+      throw new WorkerException("dbt debug Failed.");
+    }
+    dbtConfig.withDbtArguments("deps");
```

Comment:
for my own knowledge: what are these dbt arguments? is there something I can read to understand this?

Reply:
Here: 
https://docs.getdbt.com/reference/dbt-commands